configuration.h

```diff
@@ -478,6 +479,9 @@ typedef struct settings
       char path_cheat_database[PATH_MAX_LENGTH];
       char path_content_database[PATH_MAX_LENGTH];
       char path_overlay[PATH_MAX_LENGTH];
+      char path_record_config[PATH_MAX_LENGTH];
+      char path_stream_config[PATH_MAX_LENGTH];
+      char path_stream_url[PATH_MAX_LENGTH];
```

Comment:
Needs a setting under settings->recording.
Here is were you would store your twitch.tv or youtube URL.

If the URL is not defined it will fallback to stream to udp://127.0.0.1:video_stream_port